src/Pages/HomePage.css

```diff
@@ -0,0 +1,61 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+}
+/* MOBILE STYLES */
+.logo {
+  color: rgb(8, 8, 133);
+  left: 10px;
+  margin: 30px;
+}
+.block {
+  margin: 10px;
+}
+.agency-card-container {
+  display: grid;
+  justify-content: center;
+  margin-top: 40px;
+  grid-template-columns: repeat(2, 100px);
+
+  grid-column-gap: 20px;
+  border-bottom: 1px solid rgb(119, 118, 118);
+}
+/* .card-container-border {
+  border: 1px solid grey;
+} */
+.company-logo {
+  display: grid;
+  justify-content: left;
+  /* margin-left: 25px; */
+  max-width: 30px;
+}
+.agency-card-info {
+  display: grid;
+  grid-template-columns: repeat(1, 190px);
+  /* justify-content: center; */
+}
+.company-name {
+  width: 40px;
+  margin-right: 25px;
+  font-size: 14px;
+}
+.company-info {
+  display: grid;
+  justify-content: left;
+  margin-top: 15px;
+  font-size: 16px;
+  margin-bottom: 10px;
+}
+.company-info h5 {
+  margin-bottom: 5px;
+}
+span {
```

Comment:
Keep in mind that you're styling ALL span's when you do this. 

So when someone else uses a span in the rest of the website it will be styled like this. 

That means that person will need to change this and a class most likely. 

You can use tag selector sometimes but only for defining "default" styles for the entire website. The use span is usually very situational, so default don't make that much sense.